.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@
     -   id:  double-quote-string-fixer
     -   id:  end-of-file-fixer
     -   id:  flake8
-        exclude: blog/migrations
+        exclude: '^blog/migrations/|^mysite/settings.py|^mysite/wsgi.py'
```

Comment:
Because I need to import DjangoWhiteNoise after setting defaults.
>'DJANGO_SETTINGS_MODULE' environment variable must be set before importing >'whitenoise.django'